content/news/2024-10-17-newsletteroct24/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Into the EVERSE (October 2024)"
+draft: false
+date: 2024-10-17T00:00:00Z
+start_date: 2024-10-17T00:00:00Z
+end_date: 2024-10-17-T00:00:00Z
+summary: >
+    Our first newsletter is out now! 
+    Read it here.
+---
+
+**Newsletter (N<sup>o</sup> 1) October 2024**
+{{< flex-content >}}
+<--->
+{{< figure src="feature.png">}}
+{{< /flex-content>}}
+
+Our first newsletter is finally ready! It's been a busy first half of a year for us and we wanted to bring you up-to-date.
+*Into the EVERSE* is a quarterly newsletter that will summarise our activities, milestones and latest achievements.
+The first edition introduces the project itself with an interview with our project leader Fotis Psomopoulos; reports on meetings we've been to, especially the International Research Software Funders Workshop; talks about two launched services (RSQkit and the EVERSE Network); and an outlook on where research software engineering will be highlighted in the next months.
+
+
+Follow the link [here](https://preview.mailerlite.io/emails/webview/1144236/135360828853454549) or read the {{< a href=/newsletter/"Newsletter_Oct24.pdf download="Newsletter_Oct24"  >}}
```

Comment:
Further comment on the newsletter filename. They should be names in such a way as they order correctly alphabetically, so I would like ISO style here. In this case `Newsletter-2024-10.pdf`.